modules/openapi-generator/src/main/resources/csharp/libraries/generichost/api.mustache

```diff
@@ -390,6 +390,12 @@ namespace {{packageName}}.{{apiPackage}}
                     uriBuilderLocalVar.Path = urlLocalVar.AbsolutePath;
                     {{/-first}}
                     {{/servers}}
+                    {{#constantParams}}
```

Comment:
FYI @devhl-labs this change adds constant support to the path parameters via an option.

Reply:
I think instead of ToString on the enums you want ClientUtils.ParameterToString. It has logic for each of the enums.

![image](https://github.com/OpenAPITools/openapi-generator/assets/29644329/f9d824ee-0538-4e3e-9e47-a1d2a5e63d10)